build/yamls/antrea-eks-node-init.yml

```diff
@@ -72,19 +56,20 @@ spec:
               test -e /var/run/docker.sock || container_runtime="containerd"
               echo "Container runtime: $container_runtime"
 
-              # Wait for kubelet to register the file update. Default sync time is 5sec
+              # Wait for kubelet to register the file update.
+              # Default sync time is 5s so we sleep for 10s.
               # https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockershim/network/cni/cni.go#L50
-              sleep 5s
+              sleep 10s
 
               while true; do
-                  curl localhost:61679 && retry=false || retry=true
+                  curl -sS -o /dev/null localhost:61679 && retry=false || retry=true
                   if [ $retry == false ]; then break ; fi
                   sleep 2s
                   echo "Waiting for aws-k8s-agent"
               done
 
               # Fetch running containers from aws-k8s-agent and kill them
```

Comment:
@antoninbas We do restart the pods which could be scheduled with just AWS CNI when antrea is installed. So eventually all pods will be managed by antrea. If you are worried about very small window in which pod is scheduled with AWS cni and killed when antrea cni is installed, then I am OK with the change.

Reply:
This is fine when initially deploying Antrea on an EKS cluster, but there are several "edge" cases that were not accounted for initially:
- aws-node agent restart (will cause the CNI conf file to be overwritten)
- K8s Node restart, which causes all Pods to restart. aws-node will overwrite the CNI conf file, antrea-eks-node-init will not run again.
- a new Node is added to the cluster: there is no guaranteed order of execution for aws-node, antrea-agent, antrea-eks-node-init

In my last series of patches (this one being the latest), I have tried to implement a solution that covers all of these edge cases.